File: Examples/perl5/constraint/example2.pl
# Pointer example

use example;

# Allocate some memory
print "Allocating memory...\n";
$a = example::malloc(20000);
print $a,"\n";
print "Freeing memory...\n";
example::free($a);
print "Done.\n\n";

print "Generating NULL pointer...\n";
$a = undef;
print "This will generate an error\n";
print "Freeing memory...\n";
example::free($a);

File: Examples/python/constraint/example2.py
# Pointer example

from example import *

# Allocate some memory
print "Allocating memory..."
a = malloc(20000)
print a
print "Freeing memory..."
free(a)
print "Done.\n"

print "Generating NULL pointer..."
a = "NULL"
print a
print "This will generate an error"
print "Freeing memory..."
free(a)
